libs/pages/elewa/social-impact/src/lib/pages-elewa-social-impact.module.ts

```diff
@@ -12,17 +12,17 @@ import { SocialImpactCooperativeSectionComponent } from './components/social-imp
 import { SocialImpactPageComponent } from './pages/social-impact-page/social-impact-page.component';
 
 import { SocialImpactRoutingModule } from './social-impact.routing';
-import { SocialImpactOpenKnowledgeComponent } from './components/social-impact-open-knowledge/social-impact-open-knowledge.component';
 
+import { SocialImpactCallToActionSectionComponent } from './components/social-impact-call-to-action-section/social-impact-call-to-action-section.component';
+import { SocialImpactOpenKnowledgeComponent } from './components/social-impact-open-knowledge/social-impact-open-knowledge.component';
 @NgModule({
-  imports: [CommonModule, LayoutModule, BannersModule, SocialImpactRoutingModule],
-  declarations: [
-    SocialImpactPageComponent,
-    SocialImpactHeroSectionComponent,
-    SocialImpactPeopleSectionComponent,
-    SocialImpactOpenKnowledgeComponent,
-    SocialImpactCooperativeSectionComponent,
-  ],
-  exports: [SocialImpactPageComponent],
+
+  imports: [CommonModule, LayoutModule,BannersModule, SocialImpactRoutingModule],
+  declarations: [SocialImpactPageComponent, SocialImpactHeroSectionComponent,SocialImpactCallToActionSectionComponent,SocialImpactPeopleSectionComponent,SocialImpactOpenKnowledgeComponent,
+    SocialImpactCooperativeSectionComponent],
+  exports: [SocialImpactPageComponent,SocialImpactCallToActionSectionComponent],
```

Comment:
SocialImpactCallToActionSectionComponent should not be exported as its shouldn't be reusable.